modules/timeplus/src/test/java/org/testcontainers/TimeplusTestImages.java

```diff
@@ -0,0 +1,8 @@
+package org.testcontainers;
+
+import org.testcontainers.utility.DockerImageName;
+
+public interface TimeplusTestImages {
+    DockerImageName TIMEPLUS_PROTON_IMAGE = DockerImageName.parse("ghcr.io/timeplus-io/proton:latest");
+    DockerImageName TIMEPLUS_IMAGE = DockerImageName.parse("ghcr.io/timeplus-io/proton:latest");
```

Comment:
why there has 2 same images ?

Reply:
Refer to `ClickHouseTestImage.java` https://github.com/testcontainers/testcontainers-java/blob/main/modules/clickhouse/src/test/java/org/testcontainers/ClickhouseTestImages.java#L7, they want to test these two docker images repo accessibility, but we only have 1 repo,right?

Reply:
Let's not use latest tag. Let's pin the tag to a specific one, please. Also, `ghcr.io/timeplus-io/proton:latest` is used twice, unless there is a typo. Last but not least, let's avoid using `TimeplusTestImages` and use the constant or image name directly in tests